docs/modules/ROOT/pages/install-clc.adoc

```diff
@@ -1,4 +1,179 @@
-= Installing Hazelcast CLC
-:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS and Linux environments.
+= Installing the Hazelcast CLC
+:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS, Linux, and Windows environments.
+
+// See https://docs.hazelcast.com/hazelcast/5.2-snapshot/clients/clc#installing-the-hazelcast-clc
+
+{description} You can install the Hazelcast CLC, using one of the following:
+
+* Homebrew (recommended)
+* Installer script
+* Building from source
+
+== Installing on macOS
+
+The Hazelcast CLC is supported on macOS 12 or newer versions.
+
+[tabs] 
+==== 
+Installer Script::
++
+--
+macOS 12 or newer comes with cURL bundled with the operating system. You can directly run the following command.
+
+[source,bash]
+----
+curl https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/scripts/install.sh | bash
+----
+--
+
+Homebrew::
++
+. Make sure Homebrew is installed using the following command.
++
+[source,bash]
+----
+brew -v
+----
++
+If installed, you'll get its version information. Otherwise, you'll receive an error stating the command is not found; install it following the instructions https://docs.brew.sh/Installation[here].
```

Comment:
We should avoid using 'here' as link text. It isn't useful and it's not accessible for screen readers. The link text should describe the link resource. For example:

`If you see version information, Homebrew is installed. If you don't see version information, Homebrew is not installed. Install it now by following the link:https://docs.brew.sh/Installation[Homebrew documentation].

Reply:
I see you updated this example. thanks. The document still contains other links that use 'here' for the link text. Could you update those as well?